Package.swift

```diff
@@ -149,12 +149,3 @@ let package = Package(
     ),
   ]
 )
-
-#if swift(>=5.6)
```

Comment:
Should we consider just also adding a conditional check for Windows here? Swift-DocC doesn't ship in the Windows toolchain so the plugin won't be applicable there for some time.

It might be worth leaving it around for non-Xcode users in the meantime though.

Reply:
It's still unclear to me why we need a dependency on this plugin to ship docc docs outside of Xcode. To my mind, this is something SourceKit-lsp should take care of.

Reply:
It's for building and previewing documentation locally. Longer term, I agree that the plugin should probably be included in the toolchain by default (I'm not sure if sourcekit-lsp is the right place or if we should have a notion of default SwiftPM plugins) so that we have out-of-the-box support for docs. But we would need to have more discussion around that.

Reply:
The plugin can be built on Windows even if the rest of DocC cannot be supported due to the dependency on NIO.